AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -934,32 +934,32 @@ function Request-PowerShellGetModule {
     Remove-Module -Name PackageManagement -ErrorAction SilentlyContinue
 }
 
-function Request-AzureADModule {
+function Request-MSGraphModule {
     [CmdletBinding(SupportsShouldProcess, ConfirmImpact="High")]
     param()
 
     if ($PSVersionTable.PSVersion -gt [Version]::new(6,0,0)) {
         $winCompat = Get-Module -Name WindowsCompatibility -ListAvailable
     }

```

Comment:
Is WindowsCompatibility also needed for MS Graph in PowerShell versions >6.0.0? Or is that an AzureAD thing only?

Reply:
removed